templates/docs/db/fizz.md

```diff
@@ -35,7 +37,7 @@ create_table("users") {
 }
 ```
 
-It will also generate two `timestamp` columns; `created_at` and `updated_at`.
+Note that a primary key is not required. The `create_table` function will generate two `timestamp` columns; `created_at` and `updated_at`.
```

Comment:
It's not needed for fizz, but I think it's needed for buffalo (so when using buffalo it'll be created, if so it might be good to precise it). Am I wrong @stanislas-m ?

Reply:
Pop will still generate an ID, even if fizz doesn't require it. Fizz had its rule loosen because it can be used outside Pop scope, and requiring an ID is Pop rule.